README.rst

```diff
@@ -45,7 +45,7 @@ something like:
 
   app = flask.Flask(__name__)
   app.config['FLASK_HTPASSWD_PATH'] = '/path/to/.htpasswd'
-  app.config['FLASK_SECRET'] = 'Hey Hey Kids, secure me!'
+  app.config['SECRET_KEY'] = 'Hey Hey Kids, secure me!'
```

Comment:
Why the change in configuration key? It doesn't seem worth breaking backwards compatibility to me, but maybe there is a good reason. Could you elaborate on why? Thanks!

Reply:
The reason is that FLASK_SECRET doesn't exist in Flask configs, SECRET_KEY does
I don't know why FLASK_SECRET was there tbh, maybe that was your custom secret name?

Reply:
Yep, that was intentional so that it wasn't the same key used to sign cookies. It was so long ago though, that I don't fully remember why I found that useful but I would guess that it was due to a concern for having more exposure to crack the key by virtue of having signatures available to a potential attacker in cookies for a given app.

Reply:
Ohh I see... My bad never thought of that

At first, I thought it was a mistake but kinda unusual to see it everywhere for a bug 😅

Well I can revert it tomorrow if it is planned to keep both keys different.